constructor/preconda.py

```diff
@@ -15,7 +15,7 @@
 except:
     import ruamel_json as json
 
-files = 'urls', 'urls.txt', '.install.py'
+files = 'system.info', 'urls', 'urls.txt', '.install.py'
```

Comment:
@kalefranz , the file name changed below but not here! Can you please throw in a quick change to fix this. Also, double second @goanpeca to actually do some testing along the way.

Reply:
https://github.com/conda/constructor/pull/142/